core/assets/sounds/cues/LumBearJack/data.json

```diff
@@ -0,0 +1,149 @@
+{
+  "gameID": "lumBearJack",
+  "gameName": "LumBEARjack",
+  "series": "megamix",
+   "cues": [
+   {  
+   "id" : "lumBearJack/Nyah"

```

Comment:
Syntax errors everywhere. Run this json file through a validator and fix all of them.